src/scheduler.rs

```diff
@@ -44,13 +70,15 @@ pub struct ScheduledPayJoin {
 impl ScheduledPayJoin {
     pub fn new(
         wallet_amount: bitcoin::Amount,
-        channels: Vec<ScheduledChannel>,
-        fee_rate: u64,
+        batch: ChannelBatch,
     ) -> Self {
-        Self { wallet_amount, channels, fee_rate }
+        Self {
+            wallet_amount,
+            channels: batch.channels().clone(),
+            fee_rate: batch.fee_rate() }
```

Comment:
`/pj/schedule` should return this struct alongside the bip21 to display the pending transaction that the bip21 represents